ui/tests/__tests__/FacetsGrid.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import FacetsGrid from "../../src/components/facets/FacetsGrid";
+import { getFacetsList } from "../mockutils";
+
+test("Renders with input", () => {
+  const wrapper = shallow(
+    <FacetsGrid updateFacets={() => {}} facets={getFacetsList(4, 6)} />
```

Comment:
`getFacetsList(2, 2)` to make snapshots shorter and have less boilerplate.

Reply:
Done